src/app/pages/product/recommended-products/recommended-products.component.html

```diff
@@ -1,9 +1,18 @@
-<div>
-  <h3>You will maybe love</h3>
-  <div class="recommended__cards">
-    <app-recommended-product-card></app-recommended-product-card>
-    <app-recommended-product-card></app-recommended-product-card>
-    <app-recommended-product-card></app-recommended-product-card>
-    <app-recommended-product-card></app-recommended-product-card>
-  </div>
+<div class="recommended">
+    <div class="recommended__title">
+        <h2>You will maybe love</h2>
+        <button class="recommended__button" (click)="getNextRecommended()"
+                [disabled]="this.endIndex >= this.productsFromCategory.length">
+            <span>More products</span>
+            <img src="assets/images/svg/ic-arrow-right-green.svg" alt="arrow">
+        </button>
+    </div>
+    <ul class="recommended__list">
+        <li class="recommended__item-empty" *ngIf="!this.productsFromCategory.length">
```

Comment:
The same here, you will get an error if there you will receive from Back-End or somewhere else value like `null` or `undefined` like this: 

> _Can't read property `length` of `undefined`_

```suggestion
        <li class="recommended__item-empty" *ngIf="!this.productsFromCategory?.length">
```